lib/client/mfa.go

```diff
@@ -196,3 +196,45 @@ func MakeAuthenticateChallenge(protoChal *proto.MFAAuthenticateChallenge) *auth.
 
 	return chal
 }
+
+type TOTPChallenge struct {
+	QRCodeBytes []byte `json:"qrCodeBytes"`
+}
+
+// MFAAuthenticateChallenge is an MFA register challenge sent on new MFA register.
+type MFARegisterChallenge struct {
+	// U2FChallenge contains U2F register challenge.
+	U2FChallenge *u2f.RegisterChallenge `json:"u2fChallenge"`
+	// WebauthnChallenge contains webauthn challenge.
+	WebauthnChallenge *wanlib.CredentialCreation `json:"webauthnChallenge"`
+	// TOTPChallenge contains TOTP challenge.
+	TOTPChallenge *TOTPChallenge `json:"totpChallenge"`
+}
```

Comment:
Name the fields just U2F, TOTP and Webauthn, so it matches the proto?

```suggestion
// MFAAuthenticateChallenge is an MFA register challenge sent on new MFA register.
type MFARegisterChallenge struct {
	// U2F contains U2F register challenge.
	U2F *u2f.RegisterChallenge `json:"u2f"`
	// Webauthn contains webauthn challenge.
	Webauthn *wanlib.CredentialCreation `json:"webauthn"`
	// TOTP contains TOTP challenge.
	TOTP *TOTPChallenge `json:"totp"`
}
```